pkg/util/common.go

```diff
@@ -1347,22 +1304,31 @@ func DeleteCustomResourceDefinition(apiExtensionClient *apiextensionsclient.Clie
 }
 
 // WaitUntilPodsAreReady will wait for the pods to be ready
-func WaitUntilPodsAreReady(clientset *kubernetes.Clientset, namespace string, labelSelector string, timeoutInSeconds int64) (bool, error) {
+func WaitUntilPodsAreReady(clientset *kubernetes.Clientset, namespace string, labelSelector string, timeoutInSeconds int64) error {
+
+	// check before the first tick
+	ready, err := IsPodReady(clientset, namespace, labelSelector)
```

Comment:
Can you modify IsPodReady to return a single return?  Makes it simple success/failure return state.

Reply:
done